opengemini_client/client_impl.py

```diff
@@ -129,22 +132,132 @@ def query(self, query: Query) -> QueryResult:
             results = []
 
             for result in json_data.get('results', []):
-<<<<<<< HEAD
+                series_list: List[Series] = [
+                    Series(columns=series['columns'], values=series['values'])
+                    for series in result.get('series', [])
+                    if series.get('values', [])
+                ]
+                series_result = SeriesResult(series=series_list)
+                results.append(series_result)
+
+            return QueryResult(results=results)
+
+        raise HTTPError(f"Query error: {resp.status_code}, Response: {resp.text}")
```

Comment:
Instead of 'Response', a term like 'error message' should be used here.

Reply:
msg: {resp.text}